### PR TITLE
Partial/alternative solution to #64.

### PR DESCRIPTION
**Pull request by OscarL**
Added a new "Execute command and send output to Worksheet" command. Default keybinding is CMD+Shift+ENTER.

On existing Pe installations, you need to manually assign a keybinding for the new command, under:

Preferences->Keybindings->Miscellaneous

### CODE REVIEW

**Comment by waddlesplash** [comment]
Seems OK to me, I guess?

**Comment by korli** [comment]
Please rebase on master, thanks.

File: Sources/PExec.h [hunk 1]
```diff
@@ -43,11 +43,12 @@ class PErrorWindow;
 
 class PExec : public MThread {
 public:
-			PExec(PText *txt, const char *cmd, const char *wd);
+			PExec(PText *txt, const char *cmd, const char *wd, PText *dest=NULL);
```
Comment (korli) [inline Sources/PExec.h:46]:
```suggestion
			PExec(PText *txt, const char *cmd, const char *wd, PText *dest = NULL);
```

File: Sources/PText.h [hunk 2]
```diff
@@ -265,7 +265,7 @@ virtual		void FrameResized(float w, float h);
 			void JumpToFunction(const char *func, int32 offset);
 			void FindNextError(bool backward);
 
-			void ExecuteSelection();
+			void ExecuteSelection(bool outputToWorkSheet=false);
```
Comment (korli) [inline Sources/PText.h:268]:
```suggestion
			void ExecuteSelection(bool outputToWorkSheet = false);
```

**Pull request merged**